lib/fileUpload.coffee

```diff
@@ -39,7 +41,8 @@ if UploadFS?
 			onFinishUpload: ->
 				console.log arguments
 			onRead: (fileId, file, req, res) ->
-				res.setHeader 'content-disposition', 'download'
+				unless file?.type?.match(/^image\/.*/)
```

Comment:
@gmsecrieru this can be a secure issue, we are not allowing users to open images cuz users can open SVGs with untrusted code like script tags.